docs/seqcol.md

```diff
@@ -725,7 +725,7 @@ This digest allows the comparison to be pre-computed, and more easily compared.
 - inherent: false
 - collated: false
 - passthru: false
-- transient: false
+- transient: true or false (depending on the use case/implementation)
```

Comment:
The content is a duplicate of the `sequences` attribute, except for the sorting. IMO, the considerations to make this `transient` or not might include 1) storage space on server 2) performance/overhead on server of just-in-time sorting on requests, 3) whether client-side sorting is problematic for the use-case.

Reply:
@nsheff Did you review this? If so, please resolve

Reply:
yes, I think it was fine.